### PR TITLE
[go/fi] Translate go/en to Finnish

### PR DESCRIPTION
**Pull request by ComSecNinja**
I lack formal CS education in my country so some Finnish programming terms are unknown to me and I either had to (re)invent them or consult some dusty 90's programming manuals.

### CODE REVIEW

**Comment by ComSecNinja** [comment]
The translation of the Go guide is finished.

File: fi-fi/go-fi.html.markdown [hunk 1]
```diff
@@ -0,0 +1,441 @@
+---
+name: Go
```
Comment (vendethiel) [inline fi-fi/go-fi.html.markdown]: that doesn't exist

File: fi-fi/go-fi.html.markdown [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+category: language
+language: Go
```
Comment (vendethiel) [inline fi-fi/go-fi.html.markdown]: please lowercase it

**Comment by ComSecNinja** [comment]
What do you mean? This is straight from the English version. You're telling me that's wrong?

**Comment by vendethiel** [comment]
Yeah, it should probably be lowercased as well, I think we lowercase all the names

**Comment by ComSecNinja** [comment]
Go is written that way in English, German, French, Korean, Chinese and Russian versions at least. Those are literally every one I checked. Do you think it is wise to break the consistency here or fix all of these in one pull later?

**Comment by vendethiel** [comment]
you're right, we'll fix them later

**Comment by vendethiel** [comment]
Thanks!

**Pull request merged**